### ISSUE TITLE
[vuvuzela] needs rst docs (Trac #1089)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1083]
vuvuzela needs some rst docs, even if it's just python autodocs.

Migrated from https://code.icecube.wisc.edu/ticket/1089
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:35", 
    "description": "vuvuzela needs some rst docs, even if it's just python autodocs.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067215093672", 
    "component": "combo simulation", 
    "summary": "[vuvuzela] needs rst docs", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-08-06T17:02:58", 
    "milestone": "", 
    "owner": "mjl5147", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150806T17:42:19: 
* david.schultz changed milestone from "offline V15-09-00" to "IceSim 4.3"
* david.schultz changed priority from "critical" to "major"

**Comment by icecube-trac** [comment]
Trac update at 20150901T13:29:31: 
* mjl5147 changed owner from "olivas" to "mjl5147"
* mjl5147 changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150922T01:09:54: olivas changed milestone from "IceSim 4.3" to "IceSim 5.1"

**Comment by icecube-trac** [comment]
Trac update at 20170330T18:55:40: 
* olivas commented:

> In changeset 154511:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="154511"
> replaced dox with rst.  closes #1083
> ```


* olivas changed resolution from "" to "fixed"
* olivas changed status from "assigned" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:35: 
* olivas changed milestone from "IceSim 5.1" to ""
* olivas commented:

> Milestone IceSim 5.1 deleted
